src/Core/Framework/Resources/config/packages/shopware.yaml

```diff
@@ -135,6 +135,35 @@ shopware:
             - 'klar_cpid'
             - 'klar_adid'
 
+        # Cache policies (feature flag HTTP_CACHE_POLICIES)
+        policies:
+            no_cache_private:
+                private: true
+                no_cache: true
+                max_age: 0
+                s_maxage: 0
+
+            store_api.cacheable:
+                public: true
+                s_maxage: 0
+                stale_while_revalidate: 3600
+                stale_if_error: 7200
+
+            storefront.cacheable:
+                public: true
```

Comment:
isn't the default now that public is set to true? Or did we always add it like this and then rewrite the header on reverse proxy side?

I'm wondering if we can clean that up as well, so there is only this place to configure the caching policy and nothing is rewriting it anymore